chunky/src/res/se/llbit/chunky/ui/dialogs/ConfirmResetPopup.fxml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Couldn't this popup just be a confirm dialog, instead of having its own implementation?